matchms/filtering/remove_peaks_within_window.py

```diff
@@ -0,0 +1,40 @@
+import numpy
+from ..Spikes import Spikes
+from ..typing import SpectrumType
+
+
+def remove_peaks_within_window(spectrum: SpectrumType, k: int = 6,
```

Comment:
```suggestion
def remove_peaks_outside_top_k(spectrum: SpectrumType, k: int = 6,
```

Reply:
Function name seemed to not match the Explanation in the docstring to me. Feel free to propose other names.

Reply:
The name was wrongly worded. I think `keep_peaks_within_window` is what I meant to say, but `remove_peaks_outside_top_k` sounds better.